src/v2/styles/MessageInput/MessageInput-layout.scss

```diff
@@ -129,6 +129,15 @@
     width: 100%;
   }
 
+  .str-chat__stop-generating-button {
+    background-image: var(--str-chat__circle-stop-icon);
+    width: calc(var(--str-chat__spacing-px) * 30);
+    height: calc(var(--str-chat__spacing-px) * 28);
+    cursor: pointer;
+    background-color: transparent;
+    border-width: 0;
```

Comment:
background-image, border, background-color and probably cursor belong to theme scss file. Could you please move them there?

Reply:
You're absolutely right, I thought I did but it seems I forgot. Will do !